package.json

```diff
@@ -1,6 +1,5 @@
 {
   "name": "purescript-lens",
-  "version": "1.0.1",
```

Comment:
It may be standard, and this "package" isn't on npm. However, the fact remains that someone can still install this "package" if they really want to ala `npm i git://github.com/purescript-contrib/purescript-lens.git` or some such. If we're going to use `package.json` in this library, I'd prefer it to stay valid for this reason.